cmd/argocd/commands/app.go

```diff
@@ -1293,6 +1296,7 @@ func NewApplicationListCommand(clientOpts *argocdclient.ClientOptions) *cobra.Co
 	command.Flags().StringArrayVarP(&projects, "project", "p", []string{}, "Filter by project name")
 	command.Flags().StringVarP(&repo, "repo", "r", "", "List apps by source repo URL")
 	command.Flags().StringVarP(&appNamespace, "app-namespace", "N", "", "Only list applications in namespace")
+	command.Flags().StringVarP(&cluster, "cluster", "c", "", "List apps by cluster")
```

Comment:
Thanks for this @my-git9! Can you update this string to clarify that either a URL or cluster name is acceptable?

Reply:
Updated.Do you think this is ok?